homeassistant/components/hue/sensor_base.py

```diff
@@ -278,6 +288,12 @@ class GenericZLLSensor(GenericHueSensor):
     @property
     def device_state_attributes(self):
         """Return the device state attributes."""
-        return {
-            "battery_level": self.sensor.battery
-        }
+        attributes = dict()
+        attributes.update(self.sensor.config)
+        """ Removing unnecessary values."""
+        if 'pending' in attributes: del attributes['pending']
+        if 'reachable' in attributes: del attributes['reachable']
+        if 'alert' in attributes: del attributes['alert']
+        if 'ledindication' in attributes: del attributes['ledindication']
+        if 'usertest' in attributes: del attributes['usertest']
+        return attributes
```

Comment:
I'd prefer a whitelist to a blacklist - we don't want to just start including data for the sake of it, but deliberately select properties that are appropriate.